Objective-C/Internal/Replicator/CBLConflict+Internal.h

```diff
@@ -0,0 +1,31 @@
+//
+//  CBLConflict+Internal.h
+//  CouchbaseLite
+//
+//  Copyright (c) 2019 Couchbase, Inc All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+#import "CBLConflict.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface CBLConflict (Internal)
```

Comment:
If declaring (Internal) category, don't you need to add implementation with (Internal) marked?

Reply:
implementation functions are in the CBLConflict.m similar to the CBLDatabase, CBLReplicator

Reply:
i think for the sake of consistency, i should rename that to `CBLConflict ()`

Reply:
Normally when using category, I remember that we need to do implementation with category name as well. Maybe it doesn't really need to do that.